contrib/golang/filters/http/source/cgo.cc

```diff
@@ -12,7 +12,19 @@ namespace Golang {
 // thread.
 //
 
-absl::string_view copyGoString(void* str) {
+// Deep copy GoString into std::string, including the string content,
+// it's safe to use it after the current cgo call returns.
+std::string copyGoString(void* str) {
+  if (str == nullptr) {
+    return "";
+  }
+  auto goStr = reinterpret_cast<GoString*>(str);
+  return std::string(goStr->p, goStr->n); // NOLINT(modernize-return-braced-init-list)
```

Comment:
nit: `std::string{...}` and remove the uncessary NOLINT

Reply:
done, thanks.